src/app.js

```diff
@@ -107,12 +108,14 @@ io.on('connection', function(socket) {
       const currJobId = currentJob.id;
```

Comment:
Either use this variable everywhere or stick to `currentJob.id` IMO

Reply:
Oh, thanks for pointing out, I could find only a single instance of this data extraction through the object being repeated, if you find any more, please point out. :)